tests/e2e/utils/testNotesHelperUtils.ts

```diff
@@ -121,6 +121,10 @@ const clickNoteOptionCopyLinkedNoteMarkdown = () => {
   clickTestID(TestID.COPY_REFERENCE_TO_NOTE)
 }
 
+const clickNoteOptionUnboldNote = () => {
```

Comment:
That's in the description:

>  Cypress click test has been added to the newly added UI.

and you prepared for it by creating this click helper (in line with the code base). But where do you use it? See other tests in tests/e2e/integration/note.test.ts